src/browser_action/vitals.js

```diff
@@ -18,12 +18,13 @@
   let latestCLS = {};
   let enableLogging = localStorage.getItem('web-vitals-extension-debug')=='TRUE';
   let enableUserTiming = localStorage.getItem('web-vitals-extension-user-timing')=='TRUE';
+  let enableUserTimingConsole = localStorage.getItem('web-vitals-extension-user-timing-console')=='TRUE';
```

Comment:
We used the term `User Timing` specifically to mean performance.mark() and measure() since the API for that is called User Timing: https://developer.mozilla.org/en-US/docs/Web/API/Performance_API/User_timing ... its a bit odd to say UserTimingConsole.


I see that you want to use a shared name since these both use shared data.

How about something like:

- enableAttributionToConsole
- enableAttributionToUserTiming

Another alternative is:

- Enable Console [checkbox]
- Enable User Timings [checkbox]
- Durations only, or, full attribution [radio button]

Perhaps we would even have checkbox for each of the 4 metrics so you can disable specific ones in case its noisy...

Reply:
I renamed it to `consoleTables`/`enableConsoleTables`

Options table looks like this now:

<img width="486" alt="image" src="https://user-images.githubusercontent.com/10931297/231834685-34e10c8e-f4ab-4da4-9843-9394b7fc54a7.png">